lib/resource.nunjucks

```diff
@@ -138,20 +138,26 @@
                             {% for key, item in b.formParameters %}
                               {% include "./item.nunjucks" %}
                             {% endfor %}
+
+                            {% if b.properties %}
```

Comment:
You don't need that actually since there would never be a 'properties' node at the same time you have form parameters. Form parameters is a feature in 0.8 and has been replaced by types in 1.0.